db/encoding/bytes.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package encoding
```

Comment:
question: Quick check - was this package largely copy-pasted from another repository?  There appears to be lots of dead code, and not very much documentation.  If so can you please document this and give credit to the original authors within the code (not just the PR title)?